TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ultimategoal/Robot.java

```diff
@@ -70,12 +70,19 @@ public void update() {
                 } catch (Exception e) {
                     Log.d("Module", "Module couldn't update: " + module.getName());
                 }
-                if (WILL_FILE_DUMP) {
-                    module.fileDump();
-                }
             }
         }
         telemetryDump.update();
+
+        if(isStopRequested()) {
```

Comment:
Will this always be caught? Or is there a possibility that this doesn't even get called before the program ends?

Reply:
ftc stop doesn't kill the programme i don't think, so it should be fine